adapters/http.go

```diff
@@ -3,27 +3,31 @@ package adapters
 import (
 	"bytes"
 	"errors"
+	"io"
 	"io/ioutil"
 	"net/http"
 
 	"github.com/smartcontractkit/chainlink/store"
 	"github.com/smartcontractkit/chainlink/store/models"
 )
 
-// HTTPGet requires a URL which is used for a GET request when the adapter is called.
-type HTTPGet struct {
-	URL models.WebURL `json:"url"`
-	GET models.WebURL `json:"get"`
-}
-
-// Perform ensures that the adapter's URL responds to a GET request without
-// errors and returns the response body as the "value" field of the result.
-func (hga *HTTPGet) Perform(input models.RunResult, _ *store.Store) models.RunResult {
+func request(input models.RunResult, method, url string, headers http.Header, body io.Reader, contentType string) models.RunResult {
```

Comment:
I'd move this private method to lower in the file, public stuff should go at the top IMO.

Reply:
Moved to the bottom